geemap/cartoee.py

```diff
@@ -511,16 +533,16 @@ def bbox_to_extent(bbox):
 
 
 def add_gridlines(
-    ax,
-    interval=None,
-    n_ticks=None,
-    xs=None,
-    ys=None,
-    buffer_out=True,
-    xtick_rotation="horizontal",
-    ytick_rotation="horizontal",
+    ax: Union[GeoAxesSubplot, ccrs.GeoAxes],
```

Comment:
The docs failed to build. I think this one `ccrs` is causing the issue. It needs to be imported at the beginning of the module.

Reply:
thanks for information!

Reply:
i think ``GeoAxes`` was included from  [here](https://github.com/gee-community/geemap/blob/5538f61baff05c2f7b1c25cfe90ea40762ea7413/geemap/cartoee.py#LL25C2-L25C60) 

wich 
```python 
from cartopy.mpl.geoaxes import GeoAxes, GeoAxesSubplot
```

and by remove ``ccrs`` so come up with
```py
ax: Union[GeoAxesSubplot, GeoAxes],
```
following from args in docstring information
> ax (cartopy.mpl.geoaxes.GeoAxesSubplot | cartopy.mpl.geoaxes.GeoAxes): required cartopy GeoAxesSubplot object to add the gridlines to